README.md

```diff
@@ -12,7 +12,7 @@ If you're interested in using the [GraphQL API v4][], the recommended library is
 ## Usage ##
 
 ```go
-import "github.com/google/go-github/v18/github"
+import "github.com/google/go-github/github"
```

Comment:
we should update the docs to say this: 

```
Usage: 

with go modules enabled (GO111MODULE=on or outside GOPATH): import "github.com/google/go-github/v18/github"	

with go modules disable: import "github.com/google/go-github/github"
```

Then also mention that v18 and above is go1.9.x and above. 
For anything below, people can use v17.